src/protocol/command.c

```diff
@@ -47,9 +47,12 @@ protocolCommandNew(const String *command)
     MEM_CONTEXT_NEW_BEGIN("ProtocolCommand")
     {
         this = memNew(sizeof(ProtocolCommand));
-        this->memContext = memContextCurrent();
 
-        this->command = strDup(command);
+        *this = (ProtocolCommand)
+        {
+            .memContext = memContextCurrent(),
```

Comment:
In protocolCommandNew, shouldn't we be using MEM_CONTEXT_NEW() instead of memContextCurrent()?